crates/forge/tests/cli/issue_9272.rs

```diff
@@ -0,0 +1,20 @@
+
+
+forgetest!(test_unresolved_import_fails, |prj, cmd| {
```

Comment:
please put this in the `build.rs` test file and name it/format it like so: https://github.com/foundry-rs/foundry/blob/e0636b834368f79fa9302775b92057d94789d72a/crates/forge/tests/cli/test_cmd/repros.rs#L69-L121

this helps us very easily dig up context if something regresses